content/en/building/contact-management/_index.md

```diff
@@ -2,6 +2,8 @@
 title: "Contact and User Management"
 linkTitle: "Contact Management"
 weight: 4
+description: >
+  Manage contacts and users in a CHT Application
```

Comment:
It is unfortunate to have to duplicate this description text here and down in the `hero-subtitle` block.  What do you think about updating the `layout/docs` files to automatically include the description as the page subtitle (so we would not need the `hero-subtitle` block below)?  

I did some tinkering and I think all we would need to do is add this after the `<h1>{{ .Title }}</h1>` line:

```html
          <p class="not-prose hx-text-xl hx-text-gray-600 dark:hx-text-gray-400 sm:hx-text-xl">
            {{ .Page.Description }}
          </p>
```

Reply:
This is absolutely amazing! So much simpler! THANK YOU! Implemented in https://github.com/medic/cht-docs/pull/1886/commits/25de7db7c5497f1e396a555831af41faeff5c48e